From c45b294833fd46b7af4f390ddb8d35979ce11ab1 Mon Sep 17 00:00:00 2001
From: Sunita Nadampalli <sunitan@ti.com>
Date: Mon, 8 Mar 2021 18:56:44 +0530
Subject: [PATCH 5/9] dt-bindings: Add binding for img,d5520-vxd for J721E

Add the dt-binding for the img,d5520-vxd node for J721E.

Signed-off-by: Angela Stegmaier <angelabaker@ti.com>
Signed-off-by: Sunita Nadampalli <sunitan@ti.com>
Reviewed-by: Benoit Parrot <bparrot@ti.com>
Signed-off-by: Sidraya Jayagond <sidraya.bj@ti.com>
---
 .../bindings/media/img,d5520-vxd.txt          | 22 +++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/media/img,d5520-vxd.txt

diff --git a/Documentation/devicetree/bindings/media/img,d5520-vxd.txt b/Documentation/devicetree/bindings/media/img,d5520-vxd.txt
new file mode 100644
index 000000000000..6025c0841974
--- /dev/null
+++ b/Documentation/devicetree/bindings/media/img,d5520-vxd.txt
@@ -0,0 +1,22 @@
+Imagination D5520-VXD Driver
+----------------------------
+
+The IMG VXD video decode driver for the D5500-VXD is a video decoder for
+multiple video formats including H.264 and HEVC on the TI J721E family
+of SoCs.
+
+Required properties:
+- compatible: must be "img,d5500-vxd"
+- reg: physical base address and length of the registers for the D5520
+- interrupts: should contain interrupt number for the D5520
+
+Example for J721E:
+==================
+
+       d5520: video-decoder@4300000 {
+               /* IMG D5520 driver configuration */
+               compatible = "img,d5500-vxd";
+               reg = <0x00 0x04300000 0x00 0x100000>;
+               power-domains = <&k3_pds 144>;
+               interrupts = <GIC_SPI 180 IRQ_TYPE_LEVEL_HIGH>;
+       };
-- 
2.17.1

